apis/Google.Cloud.OsLogin.V1/Google.Cloud.OsLogin.V1/OsLoginServiceClient.g.cs

```diff
@@ -656,10 +656,10 @@ public static OsLoginServiceClient Create(grpccore::CallInvoker callInvoker, OsL
         /// </param>
         /// <param name="callSettings">If not null, applies overrides to this RPC call.</param>
         /// <returns>The RPC response.</returns>
-        public virtual LoginProfile GetLoginProfile(gcoc::UserName name, gaxgrpc::CallSettings callSettings = null) =>
+        public virtual LoginProfile GetLoginProfile(gcoc::PosixAccountName name, gaxgrpc::CallSettings callSettings = null) =>
```

Comment:
Are these (seemingly breaking) changes expected?

Reply:
Good question. PosixAccountName is correct as far as I'm aware - I don't know why it would previously have been generated as UserName. Will look into it and report back.

Reply:
Ooh - it looks like it's actually a bug in the generator. Here's the definition: https://github.com/googleapis/googleapis/blob/4d45a6399e9444fbddaeb1c86aabfde210723714/google/cloud/oslogin/v1/oslogin.proto#L142

Note that PosixAccountName is the *child* type - not the resource type. So it was right before.

Reply:
Yes, it was a generator bug :( It's now fixed :)